tests/components/FilterSearch.test.tsx

```diff
@@ -442,6 +442,8 @@ it('clears input when old filters are removed', async () => {
     expect(setFilterOption).toHaveBeenCalledWith({ ...deselectedFilter, selected: true });
   });
 
+  userEvent.click(searchBarElement);
```

Comment:
I think it's not really clear how adding this line tests this case. If somebody 6 months from now had to come in and change the tests due to a refactor, or if somebody accidentally triggers this test to fail, it might be hard to understand what isn't working. Is there some way we could make it clearer?

Reply:
Adding a comment is OK but a bit of a bandaid option, there should be better ways

Reply:
Hmm. Should I add something to the title of the test?

Reply:
Or I could make a new test, but it's basically the exact same thing just with clicking into the input to check for the toggle behavior

Reply:
Could you make a new test for just this case? For unit tests we generally want short tests that each only test a single thing. This makes them easier to understand and also easier to refactor in the future if we change the behavior.

Reply:
For instance this current FilterSearch test case is actually on the long side and ideally would be shorter